modules/cloud_provider_equinix_metal/examples/simple/main.tf

```diff
@@ -0,0 +1,31 @@
+# TEMPLATE: This file was automatically generated with `generate_addon_structure.sh`
+# TEMPLATE: and should be modified as necessary
+# TEMPLATE:
+# TEMPLATE: All ".tf" files are parsed at once. There is no benefit to numerically prefixed filenames. Keep all resource definitions in "main.tf".
+# TEMPLATE:
+# TEMPLATE: When main.tf becomes unwieldy, consider submodules (https://www.terraform.io/docs/language/modules/develop/structure.html) 
+# TEMPLATE: and dependency inversion (https://www.terraform.io/docs/language/modules/develop/composition.html).
+#
+
+module "terraform_equinix_kubernetes_addons" {
+  source = "../../../../"
+
+  equinix_project = var.metal_project_id
+  equinix_metro   = "SV"
```

Comment:
Without looking, I think the Terraform provider may jitter on metro = SV declared, and metro = sv state being read in.  We may have a flag to ignore case though.  Could be easier to change this to `sv` than investigate.

Reply:
(unless you know it doesn't jitter - then no change needed)

Reply:
Confirmed it doesn't jitter